Jheferson countries branch

Anonuevo-Jheferson opened the pull request:
`add a new page "countries of the world"` - b739055c8abaddd61d72fcddf6baef2e0cea73d1
`add a input box and button` - 326fd70a7828d5d2b45180dc0d4f637528c64365
`create a http request` - b142db18bd25b69ac99999c48164f0c9208d9649

alima-kailamarie reviewed (changes requested):
Hello Mr. Anoñuevo. These are your errors in some of your codes. Please be guided.

line 12, 15, 16 = IDs should be written in snake case like "main_content" See the guidelines below:
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?usp=drivesdk

In JavaScript file, line 3, 22, 23, 41, 42, 46, 57 and 80. Don't use the arrow function, change that into another function.

Anonuevo-Jheferson commented:
> Hello Mr. Anoñuevo. These are your errors in some of your codes. Please be guided.
> 
> line 12, 15, 16 = IDs should be written in snake case like "main_content" See the guidelines below: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?usp=drivesdk
> 
> In JavaScript file, line 3, 22, 23, 41, 42, 46, 57 and 80. Don't use the arrow function, change that into another function.

fixed arrow block - 9a1246d5c0316c1fcca9afbfc8526a290943f812
fixed snake case - 36e72bfc021f8f7c0562a7a8b9eada054db6f3fb

alima-kailamarie reviewed (approved):
Your files are good to go and ready to merge it to the main branch. Thank you!
